options/posix/include/bits/posix/sa_family_t.h

```diff
@@ -1,6 +1,6 @@
 #ifndef MLIBC_SA_FAMILY_T_H
 #define MLIBC_SA_FAMILY_T_H
 
-typedef unsigned int sa_family_t;
+typedef unsigned short sa_family_t;
```

Comment:
This is an ABI break.

Reply:
should I move this + the one below into abi-bits then?

Reply:
Yes.